### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by ToyeSobayo**

### CODE REVIEW

File: client/src/pages/Search.js [hunk 1]
```diff
@@ -63,6 +68,17 @@ function Search() {
           <QuerySearchInput handleSearch={handleSearch} />
         </div>
         <div className="grid md:grid-cols-4 grid-cols-2 md:gap-4 sm:gap-2 gap-1 pb-12">
+          {searchPerformed && profiles.length === 0 && !loading && (
```
Comment (Lewin-B) [inline client/src/pages/Search.js:71]: I dont know if you need those last two ampersands after the !loading check

**Review by Lewin-B — APPROVED** [review]
One little thing everything else looks good to me

File: client/src/pages/Search.js (hunk 1 above)
Comment (ToyeSobayo) [reply client/src/pages/Search.js:71]: Might've been a typo on my part

Comment (Lewin-B) [reply client/src/pages/Search.js:71]: delete rq and see if it works

Comment (ToyeSobayo) [reply client/src/pages/Search.js:71]: Nope I need them /:

Comment (arfpog) [reply client/src/pages/Search.js:71]: can we please switch from .then to await? otherwise, looks good.

**Review by arfpog — APPROVED** [review]

**Pull request merged**